boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -100,10 +100,23 @@
 			perm-read;
 			perm-write;
 			perm-secure;
+			perm-execute;
```

Comment:
I'm thinking if we should create another `"nordic,owned-memory"` region with RWSX permission that would include `cpuflpr_code_data` and keep this one as RWS. We should have enough hardware resources to do that. It would improve security a little.

Reply:
Oh, I see I caused unnecessary actions here. I forgot FLPR has a direct bridge to RAM21 bypassing MPC. So it does require `perm-execute` to execute from RAM21. All regions can be inside a single `cpuapp_ram21_region` with permissions RWS.

I think it would be good to add a comment by `cpuflpr_code_data` to clarify why it is in a region without `perm-execute` because, as we can see in this review, it is counter-intuitive.

Reply:
fixed, added note